### PR TITLE
adding breadcumb navigation to predict page

### PR DESCRIPTION
**Pull request by fazlan-nazeem**

### CODE REVIEW

File: apps/ml/site/predict/predict.jag [hunk 1]
```diff
@@ -53,7 +53,7 @@ include("../../includes/tenantAware.jag");
 		            </div>
 
 		            <div class="wr-project">
-		                <span class="title">PROJECTS \ </span><span id="nav-project"></span><span id="nav-analysis"></span><span id="nav-model"></span>
+		                <span class="title">PROJECTS \ </span><a id="nav-project-anchor" style="color:#f0f0f0"><span id="nav-project" ></span></a><a id="nav-analysis-anchor" style="color:#f0f0f0"><span id="nav-analysis"></span></a><span id="nav-model"></span>
```
Comment (cdathuraliya) [inline apps/ml/site/predict/predict.jag:56]: Please remove inline styles if possible.

Comment (fazlan-nazeem) [inline apps/ml/site/predict/predict.jag:56]: fixed with PR https://github.com/wso2/carbon-ml/pull/200

File: apps/ml/site/predict/predict.jag [hunk 2]
```diff
@@ -396,6 +396,15 @@ include("../../includes/tenantAware.jag");
             }
         });
 
+        //for navigation using breadcumb
+        if(fromCompare == 'true'){
+        	$('#nav-analysis-anchor').attr('href','../project/compare.jag?projectId=' + projectId + '&projectName=' + projectName + '&dataseId=' + datasetId);
+    	} else{
+    		$('#nav-analysis-anchor').attr('href','../analysis/analysis.jag?analysisId=' + analysisId + '&datasetId=' + datasetId + '&projectName=' + projectName + '&analysisName=' + analysisName);
+    	}
```
Comment (Kishanthan) [inline apps/ml/site/predict/predict.jag:404]: Formatting issue. "if" and "else" are not properly aligned.

Comment (fazlan-nazeem) [inline apps/ml/site/predict/predict.jag:404]: fixed with PR https://github.com/wso2/carbon-ml/pull/202

**Pull request merged**